### DISCUSSION TITLE
进入教学模式后崩溃

### DISCUSSION THREAD

**Discussion by xiaopizi19** [ejoy/vaststars#1] [General]
从log日志看是vector数组越界

**Comment by cloudwu** [comment]
游戏开发人员今天开始休假了。而这两天引擎也有开发，所以很容易出现 bug 。

一切问题要等到过年假期结束后才能看。

**Reply by cloudwu** [reply]
我试了一下，的确有 bug 。 应该是和引擎开发不同步造成的。因为前两天的版本没有问题。如果想看看，可以尝试回滚一两天的时间。

**Comment by actboy168** [comment]
已修复

**Reply by xiaopizi19** [reply]
你好，我更新了，重新编译完运行还是有崩溃。详细的崩溃日志如下，
![NqW8QeUkLS](https://github.com/ejoy/vaststars/assets/10027521/f7c855b8-ff77-44cb-b646-0464b4ac6abf)

**Reply by actboy168** [reply]
没有更新子模块

**Reply by xiaopizi19** [reply]
谢谢，更新完可以了

**Reply by hanxlman** [reply]
刚更新的代码，进入教学模式后
<img width="830" alt="image" src="https://github.com/ejoy/vaststars/assets/44710451/ce1a336e-ed29-4249-afcd-be279f86d14d">
